poc/tests/test_vdaf.py

```diff
@@ -40,6 +40,7 @@ def test_result_share_length(self):
                       num_frac_bits=4,
                       dimension=4,
                       chunk_length=150,
+                      chunk_length_norm_equality=2,
```

Comment:
Make sure the code is consistent about "squared norm equality" versus "norm equality". Either is fine with me, but consistency is important for readability.

Reply:
I stick with "norm equality".

Reply:
Rename `PineValidSquaredNormEqualityCheck` to `PineValidNormEquality`. 
The comments may still refer to "squared norm check equality", let me know you don't like this as well.

Reply:
Comments are fine, thanks